integrations/lib/multiwoven/integrations/destination/http/config/spec.json

```diff
@@ -0,0 +1,17 @@
+{
+  "documentation_url": "https://docs.multiwoven.com/integrations/destination/http",
+  "stream_type": "dynamic",
+  "connection_specification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Http",
+    "type": "object",
+    "required": ["destination_url"],
```

Comment:
Is only the destination URL required for create an HTTP connection? Most HTTP clients use an API key or access token for authentication.
@subintp

Reply:
This is a good question, I tossed around the idea of having an api key for authentication. I can certainly add it in if we require it.